### PR TITLE
DDCE-4864 - Coverage increase

### PR DESCRIPTION
**Pull request by frikit**
- fix integration-test report when reading executed tests
- increased coverage from 77% to 87%
- cleanup unused code
- fix some warnings
- fix some compiler warnings

### CODE REVIEW

File: test/services/fileupload/FileServiceSpec.scala [hunk 1]
```diff
@@ -188,7 +188,7 @@ class FileServiceSpec extends AnyFreeSpec with SpecBase with BeforeAndAfterEach
         val updatedFile = uploadedFile.copy(downloadUrl = "drafts/DRAFT000000000/foobar")
 
         val expectedAnswers = userAnswers
-          .set(UploadSupportingDocumentPage, updatedFile)
+          .set(UploadLetterOfAuthorityPage, updatedFile)
```
Comment (EdCope) [inline test/services/fileupload/FileServiceSpec.scala:191]: would it be worth keeping and adding this as separate test, for different branch?

File: app/forms/VerifyTraderDetailsFormProvider.scala [hunk 2]
```diff
@@ -31,7 +31,7 @@ class VerifyTraderDetailsFormProvider @Inject() (
         Form(
           "value" -> text("checkRegisteredDetails.error.required.agentForTrader.consent")
         )
-      case Some(details)                                              =>
+      case Some(_)                                                    =>
```
Comment (EdCope) [inline app/forms/VerifyTraderDetailsFormProvider.scala:34]: `case _` for none exhaustive?

**Review by EdCope — CHANGES_REQUESTED** [review]

File: test/services/fileupload/FileServiceSpec.scala (hunk 1 above)
Comment (frikit) [reply test/services/fileupload/FileServiceSpec.scala:191]: it has the other branch in other tests no?

Comment (EdCope) [reply test/services/fileupload/FileServiceSpec.scala:191]: Is ok. Just thought it may make it clearer at quick glance, as 2 differences in test.

**Review by EdCope — APPROVED** [review]

**Pull request merged**